packages/lib/src/vis/shared/RatioEnforcer.tsx

```diff
@@ -17,7 +17,6 @@ function RatioEnforcer() {
     const targetScale = Math.max(camera.scale.x, camera.scale.y);
     camera.scale.x = targetScale;
     camera.scale.y = targetScale;
-    camera.updateMatrixWorld();
```

Comment:
As explained, this is no longer needed now that `moveCameraTo` no longer relies on the camera's projection matrix.